smartsim/_core/entrypoints/colocated.py

```diff
@@ -201,8 +202,17 @@ def main(
     # we generally want to catch all exceptions here as
     # if this process dies, the application will most likely fail
     try:
-        process = psutil.Popen(cmd, stdout=PIPE, stderr=STDOUT)
-        DBPID = process.pid
+        hostname = socket.gethostname()
+        filename = (
+            f"{hostname}.log"
```

Comment:
What about `colo_orc_{hostname}.log`? It would help understanding what the file represents.

Reply:
Agreed